staging/src/k8s.io/pod-security-standards/OWNERS

```diff
@@ -0,0 +1,10 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
```

Comment:
suggest `sig-auth-policy-approvers` and `sig-auth-policy-reviewers`

Reply:
Do staging repos just share the root k/k OWNERS_ALIASES file?

Reply:
k/k is the canonical location, so yes